docs/docker/import_data.md

```diff
@@ -0,0 +1,59 @@
+## Import data instructions ##
+This is an example to import a sample study: `study_es_0`.
+When try to import other studies, please follow the same routine:
+1. import gene panels (if applicable, some studies may don't need to import gene panels)
+2. import data
+
+### Step 1 - Import gene panels
+
+To import gene panels for your study, please referece the example commands in [this file](example_commands.md#importing-gene-panel)
+
+Commands for importing `study_es_0` gene panels(`data_gene_panel_testpanel1` and `data_gene_panel_testpanel2`):
+```shell
+docker run -it --rm \
+    --net cbio-net \
+    -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
+    cbioportal/cbioportal:latest \
+    bash -c 'cd /cbioportal/core/src/main/scripts/ && ./importGenePanel.pl --data /cbioportal/core/src/test/scripts/test_data/study_es_0/data_gene_panel_testpanel1.txt'
+```
+
+```shell
+docker run -it --rm \
+    --net cbio-net \
+    -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
+    cbioportal/cbioportal:latest \
+    bash -c 'cd /cbioportal/core/src/main/scripts/ && ./importGenePanel.pl --data /cbioportal/core/src/test/scripts/test_data/study_es_0/data_gene_panel_testpanel2.txt'
+```
+
+### Step 2 - Import data
+
+To import data for your study, please referece the example commands in [this file](example_commands.md#importing-data)
+
+Command for importing `study_es_0` data:
+
+```shell
+docker run -it --rm --net cbio-net \
+    -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
+    cbioportal/cbioportal:latest \
+    metaImport.py -u http://cbioportal-container:8080 -s /cbioportal/core/src/test/scripts/test_data/study_es_0
+```
+
+:warning: after importing a study, remember to restart `cbioportal-container`
+to see the study on the home page. Run `docker restart cbioportal-container`.
+
+
+## Common Questions and solutions
```

Comment:
```suggestion
## Frequently Asked Questions
```
I think this is the common title for these kind of sections

This is a great idea to include btw 👍